assets/js/googlesitekit/notifications/components/common/Dismiss.js

```diff
@@ -67,7 +72,7 @@ export default function Dismiss( {
 			onClick={ handleDismiss }
 			disabled={ disabled }
 		>
-			{ dismissLabel }
+			{ isDismissalFinal ? dismissLabel : dismissLabelInitial }
```

Comment:
Can we maybe do this in the individual notifications instead? That way we only need to pass through a `dismissLabel` and this common component remains simple.